webapp/client/src/pages/InfoTaxReturnForPensionersPage.js

```diff
@@ -0,0 +1,255 @@
+    font-size: 2.25rem;
+  }
+`;
+const ParagraphHeadingText = styled.p`
+  font-size: 1.75rem;
+  @media (max-width: 320px) {
+    font-size: 1.5rem;
+  }
+`;
+const ParagraphTextLarger = styled.p`
+  font-size: 1.5rem;
+
+  @media (max-width: 320px) {
+    font-size: 1.25rem;
+  }
+`;
+
+const ParagraphTextMedium = styled.p`
+  font-size: 1.25rem;
+`;
+
+const TestBack = styled.div`
+  padding-left: var(--spacing-03);
+  padding-right: var(--spacing-03);
+  margin: auto;
+  flex: 1;
+  width: 100%;
+  max-width: var(--main-max-width);
+`;
+
+const ContentTopSpacing = styled.div`
+  margin-top: 9.375rem;
+  position: relative;
+
+  & img {
+    width: 100%;
+  }
+`;
+
+const QuestionBoxAnchorButtons = styled.div`
+  @media (max-width: 605px) {
+    display: flex;
+    flex-direction: column;
+  }
+`;
+
+const ListBox = styled.div`
+  background: var(--bg-highlight-color);
+  padding: 32px;
+  list-style-type: none;
+
+  @media (max-width: 320px) {
+    padding: 16px;
+  }
+`;
+
+export default function InfoTaxReturnForPensionersPage({ plausibleDomain }) {
+  const { t } = useTranslation();
+
+  function trans(key) {
+    return (
+      <Trans
+        t={t}
+        i18nKey={key}
+        components={{
+          statistaLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="https://de.statista.com/statistik/daten/studie/4806/umfrage/rentenanpassungen-der-bundesregierung-seit-1999/" />
+          ),
+          destatisLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="https://www.destatis.de/DE/Presse/Pressemitteilungen/2021/08/PD21_380_73111.html" />
+          ),
+          // eslint-disable-next-line jsx-a11y/anchor-has-content
+          downloadPreparationLink: <a href="/download_preparation" />,
+          incomeCalculatorLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="https://www.finanzamt.bayern.de/Informationen/Steuerinfos/Steuerberechnung/Alterseinkuenfte-Rechner/#button" />
+          ),
+          pensionersInsuranceLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="https://www.deutsche-rentenversicherung.de/DRV/DE/Rente/Allgemeine-Informationen/Besteuerung-der-Rente/besteuerung-der-rente_node.html" />
+          ),
+          eligibilityLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="/eligibility/step/welcome?link_overview=False" />
+          ),
+          registrationLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="/unlock_code_request/step/data_input?link_overview=False" />
+          ),
+          activationLink: (
+            // eslint-disable-next-line jsx-a11y/anchor-has-content
+            <a href="/unlock_code_activation/step/data_input?link_overview=False" />
+          ),
+        }}
+      />
+    );
+  }
```

Comment:
I am not so sure if I like that function or not:

- I like it because it makes everything below a little easier to read
- I do not like it because it is unclear which links are used where